frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -88,7 +88,7 @@ export const dashboardLogic = kea<dashboardLogicType>({
         updateContainerWidth: (containerWidth: number, columns: number) => ({ containerWidth, columns }),
         updateTileColor: (tileId: number, color: string | null) => ({ tileId, color }),
         removeTile: (tile: DashboardTile) => ({ tile }),
-        refreshAllDashboardItems: (tiles?: DashboardTile[]) => ({ tiles }),
+        refreshAllDashboardItems: (payload: { tiles?: DashboardTile[]; action: string }) => payload,
```

Comment:
should this be comma separated?
```suggestion
        refreshAllDashboardItems: (payload: { tiles?: DashboardTile[], action: string }) => payload,
```

Reply:
Prettier added the semicolons here over commas :(

Reply:
🤯